corehq/apps/userreports/reports/view.py

```diff
@@ -571,15 +571,11 @@ def export_table(self):
     @property
     @memoized
     def email_response(self):
-        fd, path = tempfile.mkstemp()
-        with os.fdopen(fd, 'wb') as temp:
+        with closing(StringIO()) as temp:
```

Comment:
this means it will write the entire file to memory, right? that might be a problem for large files

Reply:
That is true, but originally it was reading the entire file into memory with 
```
    return HttpResponse(json.dumps({
        'report': f.read(),
    }))
```
If using StringIO is worse than using `f.read()` then I'm happy to drop the last commit. The second-last commit cleans up old temp files, so I think it's still useful.

Reply:
i see, didn't realize it was already doing that